io.github.atari800.Atari800.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.github.atari800.Atari800
+runtime: org.kde.Platform
```

Comment:
I don't see any traces of Qt upstream. Are you sure you need the KDE runtime?

Reply:
Switched back to freedesktop.